### PR TITLE
Export attendees entries task

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

File: lib/parzival_web/templates/user_session/new.html.heex
Comment (MarioRodrigues10) [inline lib/parzival_web/templates/user_session/new.html.heex:1]: I didn't really touch in this file, i just ran `mix format` and this changes were done.

File: lib/mix/tasks/export.attendees.entries.ex [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    Enum.to_list(1..attendee_entries)
+    |> Enum.map(fn _x -> "#{attendee.name},#{attendee.email}, #{attendee_entries}" end)
+  end
+
+  defp csv_io(attendee, tasks_user, :aggregate) do
+    attendee_entries = Enum.count(Enum.filter(tasks_user, fn tu -> tu.user_id == attendee.id end))
+
+    if attendee_entries > 0 do
+      [
+        "#{attendee.name},#{attendee.email},#{attendee_entries}"
```
Comment (RuiL1904) [inline lib/mix/tasks/export.attendees.entries.ex:67]: Here it lacks the `attendee.id`

Comment (RuiL1904) [reply lib/mix/tasks/export.attendees.entries.ex:67]: I mean, it's not necessary, but you put it on the header 😅

Comment (MarioRodrigues10) [reply lib/mix/tasks/export.attendees.entries.ex:67]: Did i ?  `Id` is like a number between 1...tasks not the `attende.id`

Comment (RuiL1904) [reply lib/mix/tasks/export.attendees.entries.ex:67]: Uhh, my bad then 😅

**Review by ruioliveira02 — COMMENTED** [review]
This would be a lot nicer if it was in a page in the admin dashboard

File: lib/parzival_web/router.ex [hunk 2]
```diff
@@ -149,6 +149,7 @@ defmodule ParzivalWeb.Router do
     put "/settings", UserSettingsController, :update
     get "/settings/confirm_email/:token", UserSettingsController, :confirm_email
     get "/cv/:attendee_id", PdfController, :download_cv
+    post "/export_atteendes", CsvController, :create
```
Comment (ruioliveira02) [inline lib/parzival_web/router.ex:152]:
```suggestion
    get "/export_atteendes", CsvController, :create
```

This is for two reasons. One is if it is a GET you can access it just by pasting the URL into a browser. Second is for semantic reasons. You are not creating new information, you are simply accessing existing data.

**Review by ruioliveira02 — CHANGES_REQUESTED** [review]
image
Happens when the user is not an admin, block this route like you did in the other PR

image

Shouldn't this be reduced to one row per attendee?

**Comment by MarioRodrigues10** [comment]
> Shouldn't this be reduced to one row per attendee?

No, because each row represents 1 entry for an user , otherwise each user will have only 1 row so everyone will have the same probability.

**Review by ruioliveira02 — APPROVED** [review]

**Pull request merged**